### DISCUSSION TITLE
Lost logs from the previous day

### DISCUSSION THREAD

**Discussion by tomaszniezbedny** [Security-Onion-Solutions/securityonion#6700] [Unsupported Versions] [answered]
Hi.

I noticed something weird, every next day there is no access to logs from the previous day in Kibana. It looks like the index for the previuos day is being cleaned up. 
<img width="1486" alt="zeek" src="https://user-images.githubusercontent.com/65168865/147452693-f76f1fe6-5097-4309-9574-0ac7a259eec7.png">
Unfortunately I did not notice when exactly it started to happen. Currently 89% (836GB) of disk space is available. 
Any ideas?

Regards,

**Comment by dougburks** [comment, marked as answer]
What is the output of the following?
```
df -h
```
```
sudo du --max-depth=1 -h /nsm
```

**Reply by tomaszniezbedny** [reply]
Hi, I attach the output below:

![df](https://user-images.githubusercontent.com/65168865/147739211-a2e58467-5be7-4786-a7ee-a8125cf81cdf.png)
![du](https://user-images.githubusercontent.com/65168865/147739218-7c3301b7-69bb-4a80-b5c9-d0285e81cd59.png)

**Reply by dougburks** [reply]
Check your `log_size_limit`:
https://docs.securityonion.net/en/2.3/elasticsearch.html#deleting-indices

**Reply by tomaszniezbedny** [reply]
the current value is 98, this is calculated as GB? I mean is it 98GB ?

**Reply by dougburks** [reply]
Yes, `log_size_limit` is expressed in GB:
https://github.com/Security-Onion-Solutions/securityonion/blob/39f32a6e138ed29a4515057fec00524d8887c398/setup/so-whiptail#L596

**Reply by tomaszniezbedny** [reply]
Ok - I'll increase to 200GB, and then we will see

**Reply by tomaszniezbedny** [reply]
did not worked : (
I mean Security Onion was stuck, due to lack of storage. I removed pcap logs, reboot and data in Elastic started to appear again.
Is there a way to limit stenographer disk usage - let's say 50% ?

**Reply by dougburks** [reply]
Generally, it should work like this:
- set `log_size_limit` for your desired retention in Elasticsearch
- Stenographer will then use the remaining space until disk free space reaches 10% and it will then start purging the oldest packet capture data

Now that you've got data appearing in Elastic again you might just want to let the system run for a while and see what it looks like. You can change the `diskfreepercentage` setting for Stenographer but that's fairly uncommon.

**Reply by tomaszniezbedny** [reply]
Great, Thank you for the advice.
